src/devices/bus/centronics/epson_lx810l.cpp

```diff
@@ -42,8 +42,10 @@
  * an offset must be used to centralize the pixels.
  */
 #define CR_OFFSET    (-14)
-#define PAPER_WIDTH  1024
-#define PAPER_HEIGHT 576
+
+#define PAPER_WIDTH  1024    // 120 dpi * 8.5333 inches
```

Comment:
For constants, we like to use the C++ "static constexpr int PAPER_WIDTH = 1024;" form now, as you may have seen me converting the A2 drivers to.  This gives a little more compile-time safety checking.

Reply:
ok (going to read up on constexpr)